src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -3489,5 +3488,89 @@ GUI_TEST_CLASS_DEFINITION(test_7715) {
     GTUtilsLog::checkContainsMessage(os, ltSizeNameList, false);
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7720) {
+    GTFileDialog::openFile(os, testDir + "_common_data/realign_sequences_in_alignment/", "amino_ext.aln");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    GTUtilsMsaEditor::setMultilineMode(os, false);
+
+    auto activeWindow = GTUtilsMsaEditor::getEditor(os)->getUI()->getUI(0);
+    CHECK_SET_ERR(activeWindow != nullptr, "Sequence widget is visible");
+
+    auto wgt = GTUtilsMSAEditorSequenceArea::getSequenceArea(os, 0);
+    CHECK_SET_ERR(wgt != nullptr, "Sequence area is visible");
+
+    auto splitter = GTWidget::findSplitter(os, "name_and_sequence_areas_splitter", activeWindow);
+    CHECK_SET_ERR(splitter != nullptr, "Splitter is visible");
+
+    QSplitterHandle* handle = splitter->handle(1);
+    CHECK_SET_ERR(handle != nullptr, "MSA Splitter handle is NULL");
+
+    GTWidget::click(os, handle);
+    QPoint p = GTMouseDriver::getMousePosition();
+    int deltaMove = wgt->width() / 3 * 2;
+    p.setX(p.x() + deltaMove);
+    GTMouseDriver::press();
+    GTMouseDriver::moveTo(p);
+    GTMouseDriver::release();
+
+    GTThread::waitForMainThread();
+    GTUtilsMsaEditor::setMultilineMode(os, true);
+
+    auto count = GTUtilsMsaEditor::getEditor(os)->getUI()->getChildrenCount();
+    bool allAreVisible = true;
+    for (uint i = 0; i < count; i++) {
+        auto w = GTUtilsMsaEditor::getEditor(os)->getUI()->getUI(i);
+        CHECK_SET_ERR(w != nullptr, "Sequence widget exists");
+        allAreVisible = allAreVisible && w->isVisible();
+    }
+    CHECK_SET_ERR(!allAreVisible, "Some line of the multiline view must be hidden");
+
+    activeWindow = GTUtilsMsaEditor::getEditor(os)->getUI()->getUI(0);
+    CHECK_SET_ERR(activeWindow != nullptr, "Sequence widget exists");
+
+    wgt = GTUtilsMSAEditorSequenceArea::getSequenceArea(os, 0);
+    CHECK_SET_ERR(wgt != nullptr, "Sequence area exists");
+
+    splitter = GTWidget::findSplitter(os, "name_and_sequence_areas_splitter", activeWindow);
+    CHECK_SET_ERR(splitter != nullptr, "Splitter exists");
+
+    handle = splitter->handle(1);
+    CHECK_SET_ERR(handle != nullptr, "MSA Splitter handle is not NULL");
```

Comment:
Low level/non-documented code that should be extracted into utility method and reused by other tests.

If you do not want to create an utility method that moves MSA splitter use other options , for example move project view splitter (search for `GTMouseDriver::dragAndDrop(splitterCenter, splitterCenter + delta);`

Reply:
fixed